CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ https://openeo.org/documentation/1.0/developers/api/reference.html#operation/con
 If needed, feature flags are used to allow testing unstable features in development/production,
 without compromising stable operations.
 
+## 2023-01-20 (0.6.3a1)
```

Comment:
current version is 0.6.7a1:
https://github.com/Open-EO/openeo-geopyspark-driver/blob/125a66f9fc82ebe0812c3bf402b9ab2083e9e944/openeogeotrellis/_version.py#L1

Reply:
I overlooked that, sorry. Now corrected.